src/plugins/faultManagement/FaultManagementView.vue

```diff
@@ -21,14 +21,14 @@
  *****************************************************************************/
 
 <template>
-<div class="c-fault-mgmt">
-    <FaultManagementListView
-        :faults-list="faultsList"
-    />
-</div>
+<FaultManagementListView
+    :faults-list="faultsList"
+/>
 </template>
 
 <script>
+// WTH is the purpose of this file?
```

Comment:
Let's remove this comment. Agree that it appears a little redundant, but the `FaultManagementListView` already has a lot going on so I'm happy to retain this separation of concerns for now.